docs/Overview.md

```diff
@@ -223,31 +231,26 @@ The IS-05 schemas `receiver_transport_params_ndi.json` and `constraints_schema_r
 ```json
 [
   {
-    "interface_ip": "10.10.10.20",
-    "server_host": "10.10.10.10",
-    "server_port": 5960,
+    "machine_name": "ndi-machine-name",
     "source_name": "ndi-sender-unique-name",
-    "group_name": "camera1"
+    "source_url" : "..."
   }
 ]
 ```
 
-**interface_ip**: 
-IP address of the network interface the receiver MUST use. If set to `auto` the receiver MUST determine which interface to use.
-
-**server_host**: 
-Hostname or IP hosting the NDI server. A `null` value indicates that the Receiver has not yet been configured.
+#### machine_name
+This indicates the device name of the Native NDI Sender that is to be connected, as utilized by the NDI SDK.
 
-**server_port**: 
-Port for the NDI server. If set to `auto` the receiver MUST determine which port to use.
+#### source_name
+The name of the Native NDI Sender stream in the NDI domain that is to be connected to the Receiver. This property SHALL NOT be concatenatedwith `machine_name` in the format `machine_name(source_name)`.
 
-**source_name**: 
-The name of the NDI stream as declared by the NDI sender.
+> Informative notes: In the NDI domain, the `source_name` and `machine_name` are concatenatedin the format `machine_name(source_name)` when streams are discovered and connected, however in the `transport_params`, these properties are kept independent. 
 
-**group_name**: 
-The NDI group of the source, `null` indicates the default group.
+#### source_url 
+The URL of the sender as utilized by the NDI SDK. The contents are proprietary to the NDI SDK and SHOULD NOT be interpreted. A controller SHOULD specify this parameter with values provided by a sender's `transport_params` or the NDI SDK discovery methods. If this parameter is set to `null` or `auto`, the receiver shall determine for itself the value of `source_url`.
```

Comment:
Same comment as above. The vendor implementing the NDI Receiver is free to interpret the url. The only requirement is that the url start with "ndi://"